ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -883,7 +876,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         let unreadDotSize = TableViewCellTokenSet.unreadDotDimensions
         unreadDotLayer.bounds.size = CGSize(width: unreadDotSize, height: unreadDotSize)
         unreadDotLayer.cornerRadius = unreadDotSize / 2
-        unreadDotLayer.backgroundColor = UIColor(dynamicColor: isSelected ? tokenSet[.brandTextColor].dynamicColor : tokenSet[.selectionIndicatorOffColor].dynamicColor).cgColor
+        unreadDotLayer.backgroundColor = UIColor(dynamicColor: tokenSet[.booleanCellBrandColor].dynamicColor).cgColor
```

Comment:
Is there a reason you chose booleanCellBrandColor over brandTextColor?

Reply:
Yep, brandTextColor uses BFG1 but we need BBG1 from booleanCellBrandColor for the unread dot.